src/Tizen.NUI/src/public/AccessibilityManager.cs

```diff
@@ -503,412 +506,665 @@ private void OnFocusOvershot(IntPtr currentFocusedView, AccessibilityManager.Foc
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             return ret;
         }
 
+        /// <summary>
+        /// Generates a new focus order number which can be used to assign to views
+        /// which need to be appended to the end of the current focus order chain.
+        /// The new number will be an increment over the very last focus order number in the focus chain.
+        /// If the focus chain is empty then the function returns 1,
+        /// else the number returned will be FOLast + 1 where FOLast is the focus order of the very last control in the focus chain.
+        /// </summary>
+        /// <returns> The focus order of the view </returns>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public uint GenerateNewFocusOrder()
         {
             uint ret = Interop.AccessibilityManage.AccessibilityManager_GenerateNewFocusOrder(swigCPtr);
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             return ret;
         }
 
+        /// <summary>
+        /// Gets the view that has the specified focus order.
+        /// It will return an empty handle if no view in the window has the specified focus order.
+        /// </summary>
+        /// <param name="order"> The focus order of the view</param>
+        /// <returns> The view that has the specified focus order or an empty handle if no view in the stage has the specified focus order </returns>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public View GetViewByFocusOrder(uint order)
         {
             View ret = new View(Interop.AccessibilityManage.AccessibilityManager_GetActorByFocusOrder(swigCPtr, order), true);
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             return ret;
         }
 
+        /// <summary>
+        /// Moves the focus to the specified view.
+        /// Only one view can be focused at the same time. The view must have a defined focus order
+        /// and must be focusable, visible and in the window.
+        /// </summary>
+        /// <param name="view"> the view to be set with</param>
+        /// <returns> Whether the focus is successful or not </returns>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public bool SetCurrentFocusView(View view)
```

Comment:
looks like all the APIs are very similar to FocusManager's APIs. could you explain why?

Reply:
When Accessibility is activated, focus is required as well. Of course, it does not use FocusManager internally, and it has its own focus implementation. The reason for the same API names is to be for the naming unity in DALi, which is C# binding.